.github/workflows/build.yml

```diff
@@ -528,6 +528,7 @@ jobs:
         - "unexpectedmaker_feathers2"
         - "unexpectedmaker_feathers2_prerelease"
         - "unexpectedmaker_tinys2"
+        - "ai_thinker_esp_12k"
```

Comment:
Please put this in alphabetical order.

Reply:
i am a user use this dev board .    not AI-Thinker  。   iam Third party 。